frontend/src/composables/use-analytics.ts

```diff
@@ -11,6 +12,11 @@ import { useUiStore } from "~/stores/ui"
 export const useAnalytics = () => {
   const { $plausible, $ua, i18n } = useContext()
   const uiStore = useUiStore()
+  const featureFlagStore = useFeatureFlagStore()
+
+  onMounted(() => {
```

Comment:
This ensures that any component that uses analytics automatically syncs `localStorage` with the feature flag.